sormas-rest/src/main/java/de/symeda/sormas/rest/CaseResource.java

```diff
@@ -119,6 +119,14 @@
 		return FacadeProvider.getCaseFacade().getIndexPage(criteriaWithSorting.getCriteria(), offset, size, criteriaWithSorting.getSortProperties());
 	}
 
+	/**
+	 * 
+	 * @param criteriaWithSorting
+	 *            - The criteria object inside criteriaWithSorting the cannot be null! Use instead an empty criteria!
```

Comment:
Text correction (also in EventResource): "The criteria object inside criteriaWithSorting cannot be null. Use an empty criteria instead."